storage/src/main/java/io/atomix/storage/statistics/StorageStatistics.java

```diff
@@ -16,15 +16,19 @@
 package io.atomix.storage.statistics;
 
 import java.io.File;
+import java.lang.management.ManagementFactory;
+import com.sun.management.OperatingSystemMXBean;
```

Comment:
can we use the java.lang.management import for OperatingSystemMXBean instead?

Reply:
OperatingSystemMXBean operatingSystemMXBean = ManagementFactory.getOperatingSystemMXBean();

Reply:
there is no memory api in java.lang.management.OperatingSystemMXBean (getFreePhysicalMemorySize getTotalPhysicalMemorySize)

Reply:
only provided in com.sun.management

Reply:
could we use Runtime.getRuntime().freeMemory() and Runtime.getRuntime().totalMemory() instead?

Reply:
Runtime.getRuntime()  can only get  memory info about jvm process , not host

Reply:
seems only com.sun.management.OperatingSystemMXBean or provide a interface to user

Reply:
doesn't the jvm memory size make more sense? we could also instanceof the mxbean to make sure there is not a classcastexception, and if it's null, then fallback to runtime? @kuujo wdyt?

Reply:
In our scenario, the host memory keeps growing and jvm's memory cannot represent the host's memory

Reply:
The old API in StorageStatistics is also to get the host disk space. Maybe we don't need to think about jvm. Are there any other better suggestions?

Reply:
let's wait to hear from @kuujo

Reply:
ok

Reply:
mbean server would be another possibility 

```java
MBeanServer mBeanServer = ManagementFactory.getPlatformMBeanServer();
    Object attribute = mBeanServer.getAttribute(new ObjectName("java.lang","type","OperatingSystem"), "TotalPhysicalMemorySize");
```

Reply:
@zhaozhongguo I spoke with kuujo, please go ahead with the mbean approach.